demo/src/main/java/com/eis/geoCalendar/demo/Behaviour/EventMapBehaviour.java

```diff
@@ -0,0 +1,470 @@
+package com.eis.geoCalendar.demo.Behaviour;
+
+import android.view.View;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.FragmentManager;
+
+import com.eis.geoCalendar.app.GenericEvent;
+import com.eis.geoCalendar.demo.Behaviour.listener.OnEventCreatedListener;
+import com.eis.geoCalendar.demo.Behaviour.listener.OnEventRemovedListener;
+import com.eis.geoCalendar.demo.Behaviour.listener.OnEventTriggeredListener;
+import com.eis.geoCalendar.demo.Behaviour.listener.OnMapInitializedListener;
+import com.eis.geoCalendar.demo.Bottomsheet.BottomSheetBehaviour;
+import com.eis.geoCalendar.demo.Dialogs.AbstractAddEventDialog;
+import com.eis.geoCalendar.demo.Dialogs.AbstractRemoveEventDialog;
+import com.eis.geoCalendar.demo.Localization.GoToGoogleMapsNavigator;
+import com.eis.geoCalendar.events.Event;
+import com.eis.geoCalendar.gps.GPSPosition;
+import com.google.android.gms.maps.CameraUpdateFactory;
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.Marker;
+import com.google.android.gms.maps.model.MarkerOptions;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static android.view.View.INVISIBLE;
+
+/**
+ * Class created to manage Events on a GoogleMap and divide the code that manages the logic of
+ * interaction with the user from the activities, fragments, ecc.. code
+ * <p>

```

Comment:
Why do you put here a <p> and to other parts don't?

Reply:
It's auto generated by android studio, probably should delete it

Reply:
Oh, what do you usually do for auto generate emmm... <p>s ?

Reply:
I think it works only on top of classes when you leave a blank line.